cmake/configure/configure_2_trilinos.cmake

```diff
@@ -175,6 +181,20 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
           MESSAGE(STATUS "Module ${_optional_module} not found!")
       ENDIF()
       ENDFOREACH()
+
+      #
+      # Check for third-party libraries (tpl).
+      #
+      FOREACH(_optional_tpl MUMPS)
+      ITEM_MATCHES(_tpl_found ${_optional_tpl} ${Trilinos_TPL_LIST})
```

Comment:
We may combine the check for third-party libraries with the module check above to save some lines of code. However, I prefer to treat them separately.

Reply:
I think that's alright as you do here. I've learned what TPL stands for this way :-)

Can you indent lines 189-196 by two spaces to make it easier to read? (I know you only copied the text from above, where that is also a problem -- if you feel ambitious, also indent lines 175-182.)